background/intro.md

```diff
@@ -1,24 +1,54 @@
-them learn.  
+## Motivation
+Assessment can have a big impact on our students. While students often view the work they do in class as a means to a grade, our goal as instructors is typically that the activities we ask them to do help them learn.  
 
-We can focus the way we evaluate that work to center students learning to help
-direct their attention to their own learning.  
+If you're here, you may have noticed/felt that:
+- You want learning in your classes to come more from intrinsic motivation
+- Students approach assignments as a formula rather than a learning experience
+- You want to create solid procedural justice so that you can grade with minimum bias
+- Grading takes longer than you'd like, especially if you're providing in-depth feedback
+- You want to shift students' goal of getting a good grade to learning to their potential
+- You want to give students opportunities to try multiple times, to learn and revise, without the grading being too onerous
 
-<!-- something like Anna's first intro slide -->
+One way to help students shift mindsets is by first shifting our own. We can focus the way we evaluate that work to center students learning to help direct their attention to their own learning.  
 
 <blockquote class="twitter-tweet"><p lang="en" dir="ltr">You&#39;re not grading assignments. <br><br>You&#39;re collecting evidence to determine student progress and pointing them towards their next steps.<br><br>Make the mental switch. It matters.</p>&mdash; Tyler Rablin (@Mr_Rablin) <a href="https://twitter.com/Mr_Rablin/status/1307518756344659973?ref_src=twsrc%5Etfw">September 20, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
 
-<!-- blurbs from atl chat that may be helpful
+## Making the switch
+When starting to look into alternative forms of assessment, we may become overwhelmed by all the different terms, grading schemes, and the wealth of resources with no clear roadmap of where to start and what to do. Sometimes we may even become discouraged because we try to change too much in one attempt or because no one else in our department has attempted something like this and we feel unsupported. However, the goal of this workshop and workbook is to help you get started and have you join a community of fellow educators making the switch to center learning and Eequity in assessment.
 
-- They seem to make students approach assignments as a formula rather than a learning experience.
+### Key features of equitable, learning-focused assessment
+An incredibly brief summary of what our goals are:
+- Explicit expectations (of work **and** background knowledge)
+- Multiple attempts, to mirror learning process
+- Clear and valuable feedback
+- Unbiased assessment
+- Agency for students
+
+### Getting started
+To get started, we recommend you:
```

Comment:
```suggestion
### Getting started

```{note}
If you're joining live in the workshop, we'll guide you through these steps
```

To get started, we recommend you:
```